app/src/androidTest/java/org/liberty/android/fantastischmemo/test/ui/FavouriteTest.java

```diff
@@ -1,114 +1,115 @@
-                allOf(withId(R.id.recent_open_list),
-                        withParent(withId(R.id.viewpager))));
-        recyclerView2.perform(actionOnItemAtPosition(0, click()));
-
-        ViewInteraction appCompatImageButton3 = onView(
-                allOf(withContentDescription("Navigate up"),
-                        childAtPosition(
-                                allOf(withId(R.id.action_bar),
-                                        childAtPosition(
-                                                withId(R.id.action_bar_container),
-                                                0)),
-                                1),
-                        isDisplayed()));
-        appCompatImageButton3.perform(click());
-
-    }
-
-    private static Matcher<View> childAtPosition(
-            final Matcher<View> parentMatcher, final int position) {
-
-        return new TypeSafeMatcher<View>() {
-            @Override
-            public void describeTo(Description description) {
-                description.appendText("Child at position " + position + " in parent ");
-                parentMatcher.describeTo(description);
-            }
-
-            @Override
-            public boolean matchesSafely(View view) {
-                ViewParent parent = view.getParent();
-                return parent instanceof ViewGroup && parentMatcher.matches(parent)
-                        && view.equals(((ViewGroup) parent).getChildAt(position));
-            }
-        };
-    }
-}
+//package org.liberty.android.fantastischmemo.test.ui;
+//
+//import android.support.test.espresso.ViewInteraction;
+//import android.support.test.rule.ActivityTestRule;
+//import android.support.test.runner.AndroidJUnit4;
+//import android.test.suitebuilder.annotation.LargeTest;
+//import android.view.View;
+//import android.view.ViewGroup;
+//import android.view.ViewParent;
+//
+//import org.hamcrest.Description;
+//import org.hamcrest.Matcher;
+//import org.hamcrest.TypeSafeMatcher;
+//import org.junit.Rule;
+//import org.junit.Test;
+//import org.junit.runner.RunWith;
+//import org.liberty.android.fantastischmemo.R;
+//import org.liberty.android.fantastischmemo.ui.AnyMemo;
+//
+//import static android.support.test.espresso.Espresso.onView;
+//import static android.support.test.espresso.action.ViewActions.click;
+//import static android.support.test.espresso.action.ViewActions.scrollTo;
+//import static android.support.test.espresso.contrib.RecyclerViewActions.actionOnItemAtPosition;
+//import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+//import static android.support.test.espresso.matcher.ViewMatchers.withClassName;
+//import static android.support.test.espresso.matcher.ViewMatchers.withContentDescription;
+//import static android.support.test.espresso.matcher.ViewMatchers.withId;
+//import static android.support.test.espresso.matcher.ViewMatchers.withParent;
+//import static android.support.test.espresso.matcher.ViewMatchers.withText;
```

Comment:
android imports should be listed first, followed by third party, then java. Also group the hamcrest imports together